web-wallet/src/lib/components/GasControls/GasControls.svelte

```diff
@@ -57,7 +58,28 @@
 		}
 
 		dispatchGasChange();
+		checkGasLimits();
 	}
+
+	const dispatch = createEventDispatcher();
```

Comment:
I would have left the declaration in its original place, along with the other utility functions here.